packages/teams-js/src/public/meeting.ts

```diff
@@ -491,12 +511,14 @@ export namespace meeting {
   }
 
   /**
-   * Allows an app to request the live streaming be stopped at the given streaming url
+   * Allows an app to request that live streaming be stopped.
    *
    * @remarks
-   * Use getLiveStreamState or registerLiveStreamChangedHandler to get updates on the live stream state
+   * Use {@link getLiveStreamState} or {@link registerLiveStreamChangedHandler} to get updates on the live stream state.
+   * This API can only be used in the `sidePanel` frame context.
+   * The `meetingExtensionDefinition.supportsStreaming` field in your app manifest must be `true` to use this API.
```

Comment:
Could you link to appropriate manifest documentation? This is a great detail to include in here.

Reply:
Done!